src/audio/host-zephyr.c

```diff
@@ -403,9 +406,23 @@ static uint32_t host_get_copy_bytes_normal(struct host_data *hd, struct comp_dev
 	if (!(hd->ipc_host.feature_mask & BIT(IPC4_COPIER_FAST_MODE)))
 		dma_copy_bytes = MIN(hd->period_bytes, dma_copy_bytes);
 
-	if (!dma_copy_bytes)
-		comp_info(dev, "no bytes to copy, available samples: %d, free_samples: %d",
-			  avail_samples, free_samples);
+	if (!dma_copy_bytes) {
```

Comment:
Another suggestion. Now when an event occur twice in less than a timeout, the second notification will be lost (or reported at next event occurrence, maybe minutes later). 
better this way:
if (n_skipped>0 && is_a_timeout) log a message regardless of current  dma_copy_bytes state.

I found those message extremely useful in many debuggings, so I prefer not to loose them ;)

Reply:
@marcinszkudlinski ok, done that, it adds work to the "good" case like reading the timer, checking stuff, take a look if it's worth it